ibtk/configure.ac

```diff
@@ -94,7 +94,9 @@ dnl provided by argument)
 CONFIGURE_LIBMESH
 CONFIGURE_PETSC
 # Configure other libraries that are not themselves dependencies:
-CONFIGURE_BOOST("$ABSOLUTE_SRCDIR/contrib/boost","\$(abs_top_builddir)/contrib/boost")
+if test "$USING_BUNDLED_BOOST" = yes; then
+  CONFIGURE_BOOST("$ABSOLUTE_SRCDIR/contrib/boost","\$(abs_top_builddir)/contrib/boost")
+fi
```

Comment:
Is this correct? We already check for bundled boost in `configure_boost.m4`.

Reply:
I am not sure, but I could not get `configure` to run without adding this check.

Reply:
I will take another look when I get a few minutes.